buildSrc/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts

```diff
@@ -99,6 +99,7 @@ extraJavaModuleInfo {
         requireAllDefinedDependencies()
     }
     module("com.google.code.findbugs:jsr305", "java.annotation") {
+        mergeJar("javax.annotation:javax.annotation-api")
```

Comment:
As we compile the protobuf code now, this is needed for the `@Generated` annotation (at compile time).

Reply:
Cool that makes sense why I needed it and you did not on first PR